src/components/Header.js

```diff
@@ -2,13 +2,17 @@ import { Link } from 'react-router-dom';
 
 const Header = () => (
   <nav className="navbar">
-    <h1>Bookstore CMS</h1>
+    <h1 className="nav-header">Bookstore CMS</h1>
     <ul>
       <li>
-        <Link to="/">Books</Link>
+        <Link className="nav-link" to="/">
+          BOOKS
+        </Link>
       </li>
       <li>
-        <Link to="/categories">Categories</Link>
+        <Link className="nav-link" to="/categories">
+          CATEGORIES
+        </Link>
       </li>
     </ul>
   </nav>
```

Comment:
- [ ] While you make a great effort 😃I see that you need to increase the padding of the header to make it like the zeplin design as you see in your design the padding from both sides top and bottom is less than the header zeplin, so it will be better 😊 if you increasing the padding for the header to make it nice like the original design 💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://github.com/Jeddaa/bookstore/assets/83360636/0749464b-9199-4e2d-bb98-17c890694c25)  |![image](https://github.com/Jeddaa/bookstore/assets/83360636/540acb49-f144-4dad-9d00-b87a775de3f2) |